demisto_sdk/common/schemas/incidentfield.yml

```diff
@@ -1,4 +1,5 @@
 type: map
+#allowempty: True
```

Comment:
?

Reply:
Thats the fix, commented it for the test failure proof